css-three-column-layout/styles.css

```diff
@@ -0,0 +1,82 @@
+* {
+  box-sizing: border-box;
+  margin: 0 auto;
+}
+
+body {
+  color: white;
+  font-family: sans-serif, 'Courier New', Courier;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  max-width: 1200px;
+  margin: 0 auto;
+  background-color: rgb(25, 150, 218);
+  justify-content: center;
+  align-items: center;
+}
+
+.row {
+  display: flex;
+  width: 90%;
+  justify-content: center;
+}
+
+.column-full {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  padding-top: 60px;
```

Comment:
Same here, make this a separate class.

Reply:
This class is called `column-full`, its styling should be something like `width: 100%`. If you are applying `display: flex` to a `column` class, that is usually an indicator that the `column` should contain a `row`.